locust/test/test_zmqrpc.py

```diff
@@ -40,7 +40,8 @@ def test_client_recv(self):
         self.assertEqual(msg.node_id, 'identity')
 
     def test_client_retry(self):
-        server = zmqrpc.Server('0.0.0.0', 8888)
+        # use non-zero port this time, just to increase code coverage
+        server = zmqrpc.Server('127.0.0.1', 8888)
```

Comment:
We should change the hardcoded port `8888` to `0`.

Reply:
i changed it and then changed it back, because codecov mentioned that it reduces test coverage.

Reply:
That's odd. It's still something we should change I think. To avoid false negative failed tests when port 8888 is already in use.

Reply:
Done. Pushed to master.